gitbutler-ui/src/lib/components/AccountLink.svelte

```diff
@@ -2,25 +2,29 @@
 	import Icon from '$lib/components/Icon.svelte';
 	import type { User } from '$lib/backend/cloud';
 	import { goto } from '$app/navigation';
+	import type { Persisted } from '$lib/persisted/persisted';
 
 	export let user: User | undefined;
 	export let pop = false;
+	export let isNavCollapsed: Persisted<boolean>;
```

Comment:
Could we make this a regular boolean? We only need this writable persisted store if we want to also toggle the value (which we don't do in this file).

Reply:
I'm not sure how you would like me to deal with the type issue. Should I create a second variable in the Navigation.svelte file that is a normal boolean type that follows the Persisted boolean variable, and then propagate the value of the normal boolean?